ckanext/qdes/jobs.py

```diff
@@ -150,7 +150,7 @@ def mark_as_reviewed(datasets):
 def ckan_worker_job_monitor():
     try:
         log.info(f'Sending notification to healthchecks for CKAN worker job monitor')
-        requests.get("https://hc-ping.com/60b47f7c-7f12-4936-a9be-e5b7193b2677", timeout=10)
+        requests.get(os.environ.get('MONITOR_QDES_JOBWORKER'), timeout=10)
```

Comment:
Check if the env variable is set and has a value before trying to send get request.